lib/src/app.dart

```diff
@@ -100,6 +112,51 @@ class _AppState extends ConsumerState<Application> {
     });
 
     super.initState();
+
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      final recentSeeks = ref.watch(recentGameSeekProvider).seeks;
```

Comment:
`ref.watch` can't be used in a widget lifecycle method.

But see my above comment about refactoring in quick_actions.dart.

Code will happen in `QuickActionService`. there should be 2 parts:
- first initialisation (in `start` method) where you'd use a `ref.read(recentGameSeekProvider)` to get current value
- listening to game seek provider changes (using ref.listen) and update the quick actions accordingly